mbedtls/src/bignum/mod.rs

```diff
@@ -142,7 +142,7 @@ impl Mpi {
     pub fn as_u32(&self) -> Result<u32> {
         if self.bit_length()? > 32 {
             // Not exactly correct but close enough
-            return Err(Error::MpiBufferTooSmall);
+            return Err(Error::from(codes::MpiBufferTooSmall));
```

Comment:
I don't think I'll ask you to change all these 100 lines but you should've used `.into()` instead of `Error::from`